composable_kernel/include/utility/config.amd.hpp.in

```diff
@@ -29,9 +29,9 @@
 #endif
 
 // buffer resourse
-#if defined(CK_AMD_GPU_GFX906) || defined(CK_AMD_GPU_GFX908)
+#if CK_AMD_GPU_GFX906 || CK_AMD_GPU_GFX908
```

Comment:
I think it's better to keep this unchanged, but change line 14 to 

#if 0
#define CK_AMD_GPU_GFX906 0
#elif 0
#define CK_AMD_GPU_GFX908 0
#elif 1
#define CK_AMD_GPU_GFX1030 1
#endif

So latter, when we have online compilation, we will define one of CK_AMD_GPU_GFX